desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,129 @@
+---
+description: admin controls for desktop
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure Admin Controls
+--- 
+
+>Note
+>
+>Admin Controls is currently in [Early Access](../../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
+
+This page contains information about how administrators can configure Admin Controls to specify and lock configuration parameters to create a standardized Docker Desktop environment across the organization.
+
+Admin Controls is designed specifically for organizations who don’t give developers root access to their machines.
+
+## Prerequisite
+
+You need to [configure a registry.json to enforce sign-in](../../../docker-hub/configure-sign-in.md). For this configuration to take effect, Docker Desktop users must authenticate to your organization. 
+
+## Step one: Place the `admin-settings.json` file in the correct location
+
+Place the `admin-settings.json` file on your developers' machines in the following locations:
+
+- Mac: /Library/Application Support/com.docker.docker/admin-settings.json
+- Windows: /ProgramData/DockerDesktop/admin-settings.json
+- Linux - /usr/share/docker-desktop/admin-settings.json 
+
+By placing this file in the above protected directories, end users are unable to modify it.
+
+>Note
+>
+> It is assumed that you have the ability to push the `admin-settings.json` settings file to the locations specified above through a device management software such as [Jamf](https://www.jamf.com/lp/en-gb/apple-mobile-device-management-mdm-jamf-shared/?attr=google_ads-brand-search-shared&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhEXjayUAi8FHHv1JJitFPb47C_q_RCySTmF86twF1qJc_6GST-YDmhoCuJsQAvD_BwE).
+
+## Step two: Add the key value pairs for the admin controls you want to lock in
+
+>Note
+>
+>Some of the configuration parameters only apply to Windows. This is highlighted in the table below.
+
+The `admin-settings.json` file requires a nested list of configuration parameters, each of which must contain the  `locked` parameter. 
+If set to `true`, users are not able to edit this setting from Docker Desktop or the CLI. 
+If set to `false`, users can change this setting from Docker Desktop or the CLI by directly editing the `settings.json` file. If this setting is omitted, the default value is `false`.
```

Comment:
regarding `locked: false`, we should note that this is akin to setting a `factory default`. the logic for `locked: false` is as follows:

1. For new installs, `locked: false` will pre-populate the relevant settings in the Desktop UI

2. For machines on which Desktop is already installed and being used, `locked:false` is ignored. This is because existing users of Desktop may have already updated a setting (e.g. their proxy). In these instances, the setting created by the user will have been written to the relevant config file (e.g. `settings.json` or `daemon.json`) and respect that user preference. Unless of course the admin chooses to override by setting a `locked: true`.

Hope that makes sense ! Further context within the comments on [DKM-29](https://docker.atlassian.net/browse/DKM-29).

Reply:
Regarding: "For machines on which Desktop is already installed and being used, locked:false is ignored .. because existing users ... may have already updated a setting"

What happens for settings that were not update by the user? Is `locked: false` ignored in that case?

Reply:
My understanding from previous chat with Manu, is that for settings that were not updated by the user, `locked:false` is also ignored in that case. Details are in the comments here - https://docker.atlassian.net/browse/DKM-29.